flathub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "only-arches": [
+    "x86_64",
+    "arm"
```

Comment:
we haven't supported arm (32-bits) for over 5 years. Not sure why it's here. Or even why you need to restrict architectures.